server/src/main/java/org/elasticsearch/index/mapper/AbstractGeometryFieldMapper.java

```diff
@@ -272,14 +272,33 @@ protected Builder newBuilder(String name, Map<String, Object> params) {
         }
     }
 
-    public abstract static class AbstractGeometryFieldType<Parsed, Processed> extends MappedFieldType {
+    public abstract static class AbstractSearcheableGeometryFieldType extends MappedFieldType {
+
```

Comment:
@djptek thesis my fault but I think is more correct to call it `AbstractSearchableGeometryFieldType`?